.github/workflows/create-release.yml

```diff
@@ -205,39 +198,25 @@ jobs:
         name: current-build-image
         path: build/build.oci
 
-    - name: Generate Package Receipts
```

Comment:
By running the workflow, I got below warning `No files were found with the provided path: run-receipt.cyclonedx.json. No artifacts will be uploaded.` Is this something you can validate?

Reply:
I guess the wildcard is still needed. Should be fixed by https://github.com/paketo-buildpacks/jammy-tiny-stack/pull/174/commits/30429988efc238dde42ec9575b22e74e8a039d26.

Reply:
Do you have an active repo where this workflow is running?

Reply:
Since I have reverted the unrelated changes, we no longer need to upload the artifacts, as we use these files within the same step.